### DISCUSSION TITLE
Add an option to rename "Current changes" to "Local changes"

### DISCUSSION THREAD

**Discussion by jenn-doe** [gitkraken/vscode-gitlens#1318] [Ideas]
I don't see this question brought up in past Issues/Discussions, but figured I'd gather some thoughts before creating a Feature Request. I often find myself explaining (and reminding myself of) the difference between "Current changes" and "Incoming changes" when performing a merge/rebase. Thoughts on adding an option to call "Current changes" "Local changes"?

**Comment by eamodio** [comment]
Can you post a screenshot on where in the UI you see that? Also, I'm not sure that local vs current is any clearer -- since in plenty of cases, both sets of changes are "local".
